perllib/FixMyStreet/App/Form/Waste/Bulky/Bexley.pm

```diff
@@ -91,24 +91,24 @@ has_field parking => (
             value     => 'No',
             data_hide => '#form-parking_extra_details-row',
         },
-        {   label => 'Yes – Single Yellow Lines',
-            value => 'Yes - Single Yellow Lines',
+        {   label => 'Yes – single yellow lines',
+            value => 'Yes - single yellow lines',
```

Comment:
Could this commit be a fixup commit on the thing that set this in the first place, no need for this to be a separate commit on top.